### PR TITLE
Update revshellgen.py

### PR DESCRIPTION
**Pull request by JohnRyk**

### CODE REVIEW

**Comment by t0thkr1s** [comment]
Hi!

First of all, thank you for contributing to the project!
Can you elaborate a bit more on this commit? Correct me if I'm wrong but sys.path[0] returns an empty string which makes no sense adding it before the commands directory path.

Have a great day!

File: revshellgen.py [hunk 1]
```diff
@@ -34,7 +35,7 @@
 
 choices = ['no', 'yes']
 shells = ['/bin/sh', '/bin/bash', '/bin/zsh', '/bin/ksh', '/bin/tcsh', '/bin/dash']
-commands = sorted([command for command in os.listdir('commands')])
+commands = sorted([command for command in os.listdir(sys.path[0] + '/commands')])
```
Comment (t0thkr1s) [inline revshellgen.py:38]: sys.path[0] returns and empty string

**Review by t0thkr1s — CHANGES_REQUESTED** [review]

**Comment by JohnRyk** [comment]
Hi. Thanks for reply. When I using the script I always put it into my "kit" folder such "/home/me/mykit" and create a sort link to the script like "/usr/bin/revshellgen". So when I run the command "revshellgen" the script cannot find the "commands" folder because it dose not in the "/usr/bin" folder. So add "sys.path[0]" and it can return the absolute path when I run the "revshellgen" command.

**Comment by t0thkr1s** [comment]
Thanks for clarifying it! I reviewed and tested the code which actually makes sense and it's a nice little improvement. Adding support is always a good thing, so feel free to make contributions like this. 👍

**Pull request merged**